pom.xml

```diff
@@ -146,8 +146,7 @@
         <aspectjweaver.version>1.7.4</aspectjweaver.version>
         <casclient.version>3.2.2</casclient.version>
         <ccpp.version>1.0</ccpp.version>
-        <cernunnos.version>1.2.2</cernunnos.version>
-        <cglib.version>3.1</cglib.version>
+        <cernunnos.version>1.3.0-SNAPSHOT</cernunnos.version>
```

Comment:
I think the build is trying to tell us something important here.  Far preferable to depend only upon real releases and not snapshots, since presumably any snapshot dependencies would need to be made concrete before uPortal release, and technical debt is to be avoided. :bank: